### DISCUSSION TITLE
Dudas enunciado y dominio TPA parte 2

### DISCUSSION THREAD

**Discussion by resLotus** [dds-jv/foro#140] [TPA - 2020] [answered]
**1)** Para la interpretación del enunciado asumimos que el termino "operación de egreso" utilizado en la primer parte del TP refiere a lo mismo que el termino "egreso" en esta segunda parte y que a su vez, un "egreso" es una "compra". Es correcta esta interpretación del enunciado?

**2)** Respecto de la siguiente oración del enunciado "Cada ítem de un presupuesto debe estar asociado obligatoriamente a un egreso (compras)". Lo que entendemos que dice el enunciado es que un presupuesto tiene un conjunto de varios items y cada item esta asociado a una compra (Egreso).
¿Entonces una compra tiene que estar asociada a un único item o puede estar asociada a varios?
Pensando la situación nos resulta razonable decir "Yo puedo en una misma compra, comprar más de un tipo de item"

**Comment by roli93** [comment]
Hola, @resLotus !

1) Sí, al menos por ahora son todo lo mismo

2) Mirate mi respuesta en https://github.com/dds-jv/foro/issues/36 y avisame si siguen con dudas :)

**Comment by resLotus** [comment, marked as answer]
Hola @roli93, nos vino bien para arrancar esta segunda parte, todavía tenemos algunas que otras dudas, pero creo que podemos empezar a plantear algunas ideas y cualquier coas hablar con los ayus.
